config/config.go

```diff
@@ -24,6 +24,9 @@ type Configuration struct {
 	CacheClient HTTPClient `mapstructure:"http_client_cache"`
 	AdminPort   int        `mapstructure:"admin_port"`
 	EnableGzip  bool       `mapstructure:"enable_gzip"`
+	// GarbageCollectorLimit will serve as a fixed cost of memory that is going to be held garbage
+	// before a garbage collection cycle is triggered
+	GarbageCollectorLimit int `mapstructure:"garbage_collector_limit"`
```

Comment:
For naming, I like that GarbageCollector conveys the right intent, but I'm not sure "Limit" is a precise enough descriptor. Maybe "GarbageCollectorMinimum", "GarbageCollectorAllocationHack", "GarbageCollectorThreshold" etc...

Reply:
GarbageCollectorMemoryBallast maybe?

Reply:
`GarbageCollectorThreshold` makes the most sense to me (I even named this PR that). @VeronikaSolovei9, @SyntaxNode let me know your veredict.

Reply:
I like GarbageCollectorThreshold.